docker/llm/finetune/xpu/README.md

```diff
@@ -1,13 +1,13 @@
-## Fine-tune LLM with IPEX LLM Container
+## Finetune LLM with IPEX LLM Container
 
-The following shows how to fine-tune LLM with Quantization (QLoRA built on IPEX-LLM 4bit optimizations) in a docker environment, which is accelerated by Intel XPU.
+The following shows how to finetune LLM with Quantization (QLoRA built on IPEX-LLM 4bit optimizations) in a docker environment, which is accelerated by Intel XPU.
 
 ### 1. Prepare Docker Image
 
 You can download directly from Dockerhub like:
 
 ```bash
-docker pull intelanalytics/ipex-llm-finetune-qlora-xpu:2.1.0-SNAPSHOT
+docker pull intelanalytics/ipex-llm-finetune-xpu:2.1.0-SNAPSHOT
```

Comment:
why not use ipex-llm-finetune-qlora-xpu as image name here?

Reply:
This image supports qlora, lora and qalora etc. Qlora cannot stand for our example anymore.